### ISSUE TITLE
Crackling in Muse Sounds flute playback

### ISSUE THREAD

**Issue by OsePedro** [musescore/MuseScore#19276]
### Issue type

Muse Sounds bug

### Bug description

The .mscz file in [flute-crackling.zip](https://github.com/musescore/MuseScore/files/12514329/flute-crackling.zip) contains a single flute part that crackles every time I play it. My buffer size is set to the maximum (4096) and the audio is not loud enough for the crackles to be a result of clipping. Reducing the tempo or the volume does not fix the issue. The issue occurs both during live playback and in the exported .wav file, which I have also included in the attachment. It occurs with both the flute 1 and flute 2 Muse Sounds instruments, but not with the MS Basic flute.

The issue occurs in both the latest stable release and the latest nightly version (MuseScore version (64-bit): 4.2.0-232470304, revision: github-musescore-musescore-efb6149).

### Steps to reproduce

Play the attached .mscz file or export it as a .wav file.

### MuseScore Version

MuseScore version (64-bit): 4.1.1-232071203, revision: github-musescore-musescore-e4d1ddf

### Regression

I don't know

### Operating system

Windows 10

**Comment by OsePedro** [comment]
@cbjeukendrup I don't have to turn my volume up loud to hear it. Is it louder in the .wav file that I attached than it is in MuseScore for you? Also, if you export it as a .wav file, is the crackling as loud as it is in my .wav file?

**Comment by cbjeukendrup** [comment]
Yes, the results that I get are exactly similar to your .wav file. So perhaps it's just that I'm used to putting my volume relatively soft normally :)
